conference_helper.py

```
@@ -0,0 +1,128 @@
+    message_count = 0
+    if MESSAGES_FILE.exists():
+        with open(MESSAGES_FILE, 'r') as f:
+            message_count = len(f.readlines())
+
+    return {
+        "participants": agents,
+        "message_count": message_count,
+        "active_agents": len([a for a in agents.values() if a['status'] == 'active'])
+    }
+
+def wait_for_other_agent(my_id, expected_count=2, timeout=120):
```

Ruff (0.14.3)

Unused function argument: `my_id`

(ARG001)